buf.yaml

```diff
@@ -0,0 +1,16 @@
+version: v2
+modules:
+  - path: internal/proto
+lint:
+  use:
+    - DEFAULT
+  ignore:
+    - internal/proto/connectext/grpc/health/v1/health.proto
+    - internal/proto/connectext/grpc/reflection/v1alpha/reflection.proto
+    - internal/proto/connectext/grpc/status/v1/status.proto
+  disallow_comment_ignores: true
+breaking:
+  use:
+    - WIRE_JSON
+  except:
+    - FIELD_SAME_DEFAULT
```

Comment:
I think is similar to the lint excludes - it's new in v2, but we should probably enable it. I don't think anyone's really going to care about the test schemas, but we shouldn't change the defaults in health, reflection, or status.

Reply:
Whoops, missed that - updated.